lib/hu_board/services/slack.rb

```diff
@@ -29,7 +29,7 @@ def github_url
     def transform_moved(mash)
       {
         username: "#{mash.meta.user.login} via HuBoard",
-        icon_url: "https://avatars.githubusercontent.com/u/#{mash.meta.user.id}?s=64",
+        icon_url: "#{mash.payload.issue.user.avatar_url}s=64",
```

Comment:
https://api.github.com/users/dahlbyk has `"avatar_url": "https://avatars.githubusercontent.com/u/133987?v=3"`, but I don't think we should assume the URL will always have a query string.